rules/test-title.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+/* eslint quote-props: [2, "as-needed"] */
+module.exports = function (context) {
+	var ifMultiple = context.options[0] === 'if-multiple';
+	var testCount = 0;
+
+	var ava = createAvaRule();
+	return ava.merge({
+		CallExpression: function (node) {
+			if (!ava.isTestFile || !util.isTestFunctionCall(node.callee)) {
```

Comment:
Better to check node equality.
```js
if (!ava.isTestFile || ava.currentTestNode !== node) {
```
Since `if (!ava.isTestFile || !ava.currentTestNode) {` matches all CallExpressions inside `test` function.